src/guides/v2.4/install-gde/prereq/mysql.md

```diff
@@ -2,7 +2,7 @@
 group: installation-guide
 title: MySQL
 redirect_from:
-  - guides/v2.3/install-gde/trouble/tshoot_mysql_table-open-cache.html
+  - guides/v2.4/install-gde/trouble/tshoot_mysql_table-open-cache.html
```

Comment:
Remove the redirect. The file has never existed in 2.4.

Reply:
Fixed